histomicstk/cli/NucleiDetection/NucleiDetection.py

```diff
@@ -275,6 +275,10 @@ def main(args):
     else:
         process_whole_image = False
 
+    # Provide default value for tile_overlap
+    tile_overlap = args.tile_overlap_value if args.tile_overlap_value != - \
```

Comment:
This line break halfway through a number makes it very hard to see what is going on.  Better would be
```
    tile_overlap = args.tile_overlap_value
    if tile_overlap == -1:
        tile_overlap = (args.max_radius + 1) * 4
```
If you really want to use the ternary, it'd still be better to do
```
    tile_overlap = (
        args.tile_overlap_value if args.tile_overlap_value != -1 else
        ((args.max_radius + 1) * 4))
```